package.json

```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "lint": "xo",
     "fix": "npm run lint -- --fix",
-    "mocha": "_mocha --timeout 10000 test",
+    "mocha": "_mocha --timeout 30000 test",
```

Comment:
Can you revert this back to 10 and bump the specific test case to 30s?

Reply:
I reverted the setting.